rosidl_typesupport_introspection_cpp/include/rosidl_typesupport_introspection_cpp/MessageIntrospection.h

```diff
@@ -6,7 +6,7 @@
 
 #include <rosidl_generator_c/message_type_support.h>
 
-#include "impl/visibility_control.h"
+#include "visibility_control.h"
```

Comment:
That's what I did in 7cfe203295bd3b76bd2a67be2cf831e02d8ae1a8, but that doesn't make the `include/${PROJECT_NAME}/impl` obsolete, there are still the `get_type_support_handle_impl.hpp` headers in there:

https://github.com/ros2/rosidl_dds/tree/fix_get_type_support_handle/rosidl_typesupport_introspection_cpp/include/rosidl_typesupport_introspection_cpp/impl/rosidl_generator_cpp